code/lib/cli-storybook/src/automigrate/fixes/renderer-to-framework.ts

```diff
@@ -0,0 +1,244 @@
+import { readFile, writeFile } from 'node:fs/promises';
```

Comment:
Don't forget it to add it to the index file. Also please create a canary and test it once in a project of your choice before merging.